docker/runs.sh

```diff
@@ -53,11 +53,11 @@ args=(
   --rpc-port 8899
   --bpf-program metaqbxxUerdq28cj1RbAWkYQm3ybzjb6a8bt518x1s /so/mpl_token_metadata.so
   --bpf-program BGUMAp9Gq7iTEuizy4pqaxsTyUCBK68MDfK752saRPUY /so/bubblegum.so
-  --bpf-program Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS /so/gummyroll_crud.so
```

Comment:
Also I think we need to remove gummyroll_crud from the Anchor.toml.  If its working for you now its probably because the old .so and idl files are still there from previous.